### PR TITLE
node-fs-writefile - Chivy Chia

### PR DESCRIPTION
**Pull request by tinachia**
image

### CODE REVIEW

File: node-fs-writefile/note-to-self.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const input = `${process.argv[2]}`;
```
Comment (Zhiwutian) [inline node-fs-writefile/note-to-self.js:3]: don't forget the newline character here!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: node-fs-writefile/note-to-self.js (hunk 1 above)
Comment (tinachia) [reply node-fs-writefile/note-to-self.js]: correction made. 
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**